alacritty/src/config/window.rs

```diff
@@ -61,6 +61,9 @@ pub struct WindowConfig {
 
     /// System decorations theme variant.
     decorations_theme_variant: Option<Theme>,
+
+    /// Window level.
+    pub window_level: Option<WindowLevel>,
```

Comment:
What is the difference between `None` and `WindowLevel::Normal`? Either `WindowLevel::Normal` should be removed, or the option. Otherwise this seems redundant.